### ISSUE TITLE
NVDA does not read equations properly in MS Word equation editor

### ISSUE THREAD

**Issue by msviolafangirl** [nvaccess/nvda#14943]
### Steps to reproduce:
1. Open a blank word document.
2. Press alt+equals to enter the equation editor. It says "type equation here" on the braille display, but it is not announced in speech. The same problem occurs with Narrator, only JAWS announces this correctly.
3. Type a simple math equation, such as 1/2 and press enter.
4. Press up arrow to locate the math equation.

### Actual behavior:
NVDA reads the equation as "one two". On the braille display, it is simply written as "1 2" with no fraction symbol in between.

### Expected behavior:
NVDA should read this equation as "one half". Narrator reads the equation as "one half", and JAWS not only reads the equation properly, but indicates at the beginning that the equation is math content, saying something like "math content one half". Try this with any other equation involving math symbols and you'll get similar results.

#### NVDA installed/portable/running from source:
Installed.

#### NVDA version:
2023.1

#### Windows version:
Windows 10, build 19045

#### Name and version of other software in use when reproducing the issue:
Word 365

**Comment by seanbudd** [comment]
Have you installed MathCAT or Mathplayer?

https://addons.nvda-project.org/addons/MathCAT.en.html

Math doesn't work well without one of these add-ons

**Comment by msviolafangirl** [comment]
I have MathCAT installed and the problem persists.
